addons/sourcemod/scripting/tfgo.sp

```diff
@@ -278,6 +280,19 @@ public void OnClientDisconnect(int client)
 	}
 }
 
+public Action OnTakeDamage(int &attacker, int &inflictor, float &damage, int &damagetype, int &weapon, float damageForce[3], float damagePosition[3], int damagecustom)
+{
+	if (tfgo_all_weapons_can_headshot.BoolValue)
```

Comment:
Likely need more checks, whenever if damage taken is from enemy with valid weapon, or from environmental/fall damage

Reply:
This was meant to be a draft PR, whoops. Thanks anyway.